documentation/design-docs/dotnet-tools.md

```diff
@@ -404,38 +406,25 @@ OPTIONS
 
 COMMANDS
 
-    collect   Capture one or more dumps (core files on Mac/Linux) from a process
+    collect   Capture dumps from a process
     analyze   Starts an interactive shell with debugging commands to explore a dump
 
 COLLECT
 
     dotnet-dump collect -p|--process-id <pid>
-                        [-h|--help]
-                        [--interval-sec <seconds>]
-                        [--number <number_of_dumps>]
-                        [-o|--output <output_dump_path>]
-                        [--type <dump_type>]
+                       [-h|--help]
+                       [-o|--output <output_dump_path>]
+                       [--type <dump_type>]
 
     Capture one or more dumps (core files on Mac/Linux) from a process
 
     -p, --process-id
-        The process to collect dumps from
-
-    -h, --help
-        Show command line help
-
-    --interval-sec
-        The number of seconds to wait between collecting each dump. Defaults to 10 seconds if not specified.
+        The the process to collect a memory dump.
```

Comment:
```suggestion
        The process to collect a memory dump from.
```

Reply:
Are you sure we want to end the sentence with a preposition?

Reply:
No I'm not sure : ) I guess "The process from which to collect a memory dump" is more kosher.

Reply:
That seems awkward (but proper) too. I think I'm going with your original ending with "from." change (since I've already made it).